BE | Display of Top 10 Active Members

loreen-rialmm opened the pull request:
## 🛠️ What was done:
- Modified admin/index.php, assets/js/admin.js, assets/php/processes/admin/index.php for the loading of top 10 active members

## 📸 Screenshot (if UI changes):
<img width="1802" height="645" alt="image" src="https://github.com/user-attachments/assets/5d0f3b8c-4a12-4493-8bab-dbce13ae3bb5" />

## ✅ Checklist:
- [ ] Code compiles with no errors
- [ ] Tested locally
- [ ] Ready for code review

## 📎 Related Issues / Task Link:
- Closes #122

binibiningjenna reviewed (changes requested):
based on lira's checking

loreen-rialmm commented:
## 🛠️ What was done:
- Check the files changed for the modified files, but overall I implemented the Top 10 Active Members, The Dashboard Statistics for the Admin Side, and the Workout Statistics for the User Side, Added a Renew button for the Membership Page Admin Side

## 📸 Screenshot (if UI changes):
Top 10 Active Members:
<img width="1807" height="619" alt="image" src="https://github.com/user-attachments/assets/df06995c-6fdd-413b-962c-bdd9223e25ff" />

Dashboard Statistics:
<img width="1820" height="462" alt="image" src="https://github.com/user-attachments/assets/7c88d412-8d19-44c7-b6a3-dd17f009c80b" />

Workout Statistics:
<img width="1914" height="746" alt="image" src="https://github.com/user-attachments/assets/d679bc5b-747b-4a75-9efa-8716db5f128c" />

Renew Membership Button:
<img width="1028" height="699" alt="image" src="https://github.com/user-attachments/assets/e420743d-4ca6-4ac1-852c-243b5e9d6652" />


## ✅ Checklist:
- [ ] Code compiles with no errors
- [ ] Tested locally
- [ ] Ready for code review

## 📎 Related Issues / Task Link:
- Closes #120
- Closes #121
- Closes #122

binibiningjenna reviewed (changes requested):
### USER CHART

### 01
- Yung sa pag gamit ng calendar ay multiple select kasi sa gym ay 2-3 workout types per day (in real life), tapos nag try ako mag lagay ng multiple workout and ganito yung labas. Much better sana if macount yung tatlo as three different workout type hindi as one hehe. Click CTRL tas click yung workout para mag multiple select kasi yung calendar natin ay design para may multiple wokrout per day 
<img width="791" height="444" alt="image" src="https://github.com/user-attachments/assets/c1d66ad4-1131-4b4f-9ece-a15e66e0ca3b" />

- ganito yung multiple select usage
<img width="1617" height="563" alt="image" src="https://github.com/user-attachments/assets/d5b761c4-93fe-4714-be43-c36bf1f5dc89" />


### 02
- nag try ako mag lagay ng workout sa year 2024 using buttons tapos nilagay ko yung workout ng AUguest 20, 2024 ganon yung hinfi current year. Tapos nong na-add ko na sya, nag try ako switch year sa workout statistics from 2025 pinili ko yung 2024, pero hindi lumabas yung chart huhu like napunta sya sa user dashboard
<img width="754" height="299" alt="image" src="https://github.com/user-attachments/assets/d4b38fa3-7532-4f48-8df1-cff8c11aed29" />


### ADMIN DASHBOARD

### 03
- Suggestion lang don sa part ng "TOP 10 ACTIVE MEMBERS", much better siguro may current month na dynamic mag change depende sa current date "JULY TOP 10 ACTIVE MEMBERS". Then mawawala na yung "....of this month" sa workout column. Magiging "Total Workouts" at "Total Points".  Pachange din sana to UPPERCASE yung mga column title para same format sa lahat ng table.
- then yung sa concern sa points sa if kaya rin mag reset, not totally hard reset sa points column ng user. Parang manipulate lang yung lalabas, lalabas yung points ng July based sa attendance. Kasi yung points ay kapag nag attendance doon lang nag kakapoints.
<img width="1738" height="474" alt="image" src="https://github.com/user-attachments/assets/c72ba5b1-3d6e-4252-a364-ea33514d1cb4" />


### ADMIN MEMBERSHIP 

### 04
- pa-add nalang loreen ng changes sa part na ito yung sa ID (user_membershipID) na suggestion ni yeye para daw same lahat ng table format. Then if nabago nasya, pachange nalang din nong "NO MEMBERSHIP DATA AVAILABLE" to center since kapag nag add ng new column ay ma-addjust sya.
<img width="1803" height="670" alt="image" src="https://github.com/user-attachments/assets/9dff528a-6769-44e1-b92f-f86a9c73b373" />

### 05
- regarding sa renew ay nag try ako mag renew kahit na may current subscription ako, nag renew pa rin sya. much better siguro if bawal mag renew hanggat may active membership hehe like kahit i-toast nalang din yung notif nya na may currenly ay active pa sya, yung mga inactive lang yung pwede mag renew para hindi sana mag doble doble yung membership. 

### 06
- naparami yung select users HAHAHAH isa lang siguro matira hehe tatlo kasi syaa pati doon sa select a plan na part 
<img width="1003" height="314" alt="image" src="https://github.com/user-attachments/assets/f6104c42-35c1-4f52-aa16-97f71d1a6462" />
<img width="899" height="162" alt="image" src="https://github.com/user-attachments/assets/cbf6eec8-26e8-4950-b4ca-b43a3ad70dd7" />

### 07
- pa-uppercase lahat ng column title para same format sa lahat ng table
<img width="1773" height="84" alt="image" src="https://github.com/user-attachments/assets/fdb99eb9-2a37-4b7c-b82d-94665ac71651" />


### NOTE
- plus plus yung nakalimutan natin na Inactive users HAHAHAH

loreen-rialmm commented:
## 01 - Multiple Workouts 
<img width="700" height="558" alt="image" src="https://github.com/user-attachments/assets/75214657-5794-4b97-a513-314f578619fd" />
<img width="658" height="326" alt="image" src="https://github.com/user-attachments/assets/706fb81b-75bd-48e3-b924-dc268a3ac88f" />
<img width="1852" height="588" alt="image" src="https://github.com/user-attachments/assets/0bf4d987-5889-4f2f-a4d5-0dfa6f173a6d" />
<img width="861" height="608" alt="image" src="https://github.com/user-attachments/assets/11807815-60f4-46d3-b161-0936a6b06da3" />

## 02 - 2024 Workout Calendar Thing
<img width="1654" height="714" alt="image" src="https://github.com/user-attachments/assets/e414681a-e437-4429-87dd-a1e6baf198a5" />
<img width="1664" height="651" alt="image" src="https://github.com/user-attachments/assets/a5e4f882-2da0-44e5-b0e8-7d1b91c6bb60" />
<img width="1695" height="767" alt="image" src="https://github.com/user-attachments/assets/92de21ee-b1d6-4841-9b3e-f1a975b76823" />

## 03 - Month Top 10 
<img width="1792" height="671" alt="image" src="https://github.com/user-attachments/assets/58c7209b-abbe-464f-b4a7-e0508b02341e" />

for the user side, the points still accumulates as per the dashboard below indicates:
<img width="1236" height="355" alt="image" src="https://github.com/user-attachments/assets/bc58cc97-5a38-4791-8847-ec660b3097da" />

## 04 - ID for the Membership
<img width="1785" height="673" alt="image" src="https://github.com/user-attachments/assets/a19e91b5-c989-4861-ad9e-53fefff8ddb2" />

then for the NO DATA is na-adjust ko na rin siya to be centered:
<img width="1236" height="226" alt="image" src="https://github.com/user-attachments/assets/4cddc167-29d8-49c3-8795-9637db00cb26" />

## 05 - No Renewal if Active 
<img width="1061" height="707" alt="image" src="https://github.com/user-attachments/assets/5ec3eb69-3553-49af-b3d2-fc35013b3c0b" />
<img width="1919" height="925" alt="image" src="https://github.com/user-attachments/assets/f2d70d21-c537-414c-a6fb-cb2282a7ba61" />

## 06 - Remove Redundancy of texts
<img width="1101" height="740" alt="image" src="https://github.com/user-attachments/assets/86683cfe-c60d-45e5-875b-6a885df0e9d4" />

i don't think nareremove yung sa select user na part below ng search function part since if anong gawin ko sa label is nangyayari rin mismo dun sa below ng search.
<img width="1210" height="704" alt="image" src="https://github.com/user-attachments/assets/639fd383-626c-4724-9083-eeb6fc8237f9" />

## 07 - Uppercase of Column Titles
<img width="1802" height="687" alt="image" src="https://github.com/user-attachments/assets/52e2e480-bf6d-46d6-b0b2-ecf241cdac1b" />

## 08 - Inactive Users for the Dashboard Statistics
<img width="1788" height="498" alt="image" src="https://github.com/user-attachments/assets/5ec6ce43-b80b-4d06-8b8f-e37044d9001e" />

binibiningjenna reviewed (changes requested):
### USER WOKROUT

### 01
- nabago yung functionality ng workout calendar huhu kasi pag nag a-add ako nag hihiwa-hiwalay na yung entries ko. pa-retain nalang nong original na kapag nag add ng marami ay ganito "Chest, Leg, Shoulder" kasi ayun yung concept ng sa calendar. Then para makuha mo yung mga workout ay gamit ka nalang ng explode para makuha mo isa-isa yung chest, leg, shoulder para sa workout statistics chart. Walang mababago sa workout calendar.

### 02
- sa workout statistics naman na part, kapag nag change ako ng dropdown ay pag ka refresh ay nabalik sa pinaka taas ng page, much better siguro if pag ka click nandon pa rin sya sa workout statistics na area kagaya nong sa part ni yeye sa leaderboard, pagkaclcick  ay nandon pa rin sya sa area.

### RENEWAL PAGE
 
### 03
- Natanggal ko na yung Select User na nag duplicate ito yung code tapos pa-apply nalang
<img width="911" height="312" alt="image" src="https://github.com/user-attachments/assets/983d949c-1a7f-4190-a783-01a4a52d20b0" />

### 04
- plus pa-apply nalang din nitong code, wala kasi syang membership plan na label nilagyan ko lang hehe
<img width="821" height="315" alt="image" src="https://github.com/user-attachments/assets/a1d8c45b-3845-4e45-ad76-a059bace85be" />

loreen-rialmm commented:
## 01 - USER WORKOUT
<img width="1658" height="720" alt="image" src="https://github.com/user-attachments/assets/62e39775-a7f8-4b06-8d89-c4c4c21adb72" />
<img width="838" height="533" alt="image" src="https://github.com/user-attachments/assets/1068b5cb-8ef4-463a-a5dd-9d5ba4066422" />

## 02 - YEAR REFRESH THING
<img width="1744" height="618" alt="image" src="https://github.com/user-attachments/assets/6fc34b5f-708c-4b8b-a789-e4e1011b399c" />
<img width="1911" height="951" alt="image" src="https://github.com/user-attachments/assets/37214659-1053-4ab0-a013-96d57e8a17b8" />

## 03 AND 04
<img width="948" height="636" alt="image" src="https://github.com/user-attachments/assets/744f2fc1-f26d-4080-aeea-b9b592bd34b5" />
